pkg/network/server_config.go

```diff
@@ -97,28 +100,36 @@ func NewServerConfig(cfg config.Config) (ServerConfig, error) {
 	if err != nil {
 		return ServerConfig{}, fmt.Errorf("failed to parse addresses: %w", err)
 	}
+
+	txsBroadcastBatchTime := cfg.ApplicationConfiguration.P2P.BroadcastTxsBatchTime
+	if txsBroadcastBatchTime == 0 {
+		// 5% of block time
+		txsBroadcastBatchTime = cfg.ProtocolConfiguration.TimePerBlock / 20
```

Comment:
@roman-khimov, I'm starting to thing that we may need to specify the *ratio* instead of particular time in the node's config if we'd like this setting to be fully adaptive. Do we need this?

Reply:
5% can also be capped at 50ms, 750ms _looks_ like a bit too much for mainnet.